### ISSUE TITLE
JWindow and JToolTip How to achieve backdrop transparent ?

### ISSUE THREAD

**Issue by Link-Kou** [JFormDesigner/FlatLaf#705]
<img width="233" alt="image" src="https://github.com/JFormDesigner/FlatLaf/assets/22113151/242f36d0-8840-4b2a-8ae9-a575cbe66ee2">

macOS：  JWindow and JToolTip  backdrop not transparent

**Comment by Link-Kou** [comment]
<img width="705" alt="image" src="https://github.com/JFormDesigner/FlatLaf/assets/22113151/fedf6051-a194-4e24-8bdb-a6ec2cdbc360">
<img width="710" alt="image" src="https://github.com/JFormDesigner/FlatLaf/assets/22113151/8161e8a1-2829-45a1-bd07-f724d0014f4c">

JDK version：JDK17 

Yes, when I open the style, JWindow has an extra background color.

**Comment by DevCharly** [comment]
What does it look like without FlatLaf?
E.g. with Metal or Aqua L&F.

**Comment by Link-Kou** [comment]
When there is no flatlaf, the background can be transparent normally

**Comment by Link-Kou** [comment]
![2023-08-03_09-05-40 (2)](https://github.com/JFormDesigner/FlatLaf/assets/22113151/4be4ea13-8b2a-42ed-bf32-9684ad980fc9)

@DevCharly

**Comment by DevCharly** [comment]
Finally, I realized what you mean 😔 : a JWindow with transparent background is not shown transparent on macOS

Test case to reproduce:

~~~java
import java.awt.Color;
import javax.swing.*;
import javax.swing.border.LineBorder;
import com.formdev.flatlaf.FlatLightLaf;

public class WindowTest
{
    public static void main( String[] args ) {
        SwingUtilities.invokeLater( () -> {
            FlatLightLaf.setup();

            JLabel label = new JLabel( "text" );
            label.setBorder( new LineBorder( Color.red ) );

            JWindow window = new JWindow();
            window.setBackground( new Color( 0, 0, 0, 0 ) );
            window.add( label );
            window.setBounds( 50, 50, 200, 200 );
            window.setVisible( true );
        } );
    }
}
~~~

On macOS this shows gray window with drop shadow:

![image](https://github.com/JFormDesigner/FlatLaf/assets/5604048/469c163f-6d35-45c4-8e1a-bf6d587b3017)

On Windows it works and does not show window border or background:

![image](https://github.com/JFormDesigner/FlatLaf/assets/5604048/ecedd6d6-5f97-4f27-b5bc-7ff3020ce874)

The reason seems to be that FlatLaf sets background color for root pane here:

https://github.com/JFormDesigner/FlatLaf/blob/bb636bac3f270eb01bcc6db891b50fd26e82d45d/flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatRootPaneUI.java#L139


To workaround this issue make the root pane also transparent:

~~~java
_jWindow.getRootPane().setBackground(new Color(0, 0, 0, 0));
~~~

Anyway, I'll fix this issue soon...

**Comment by DevCharly** [comment]
fixed in latest `3.4-SNAPSHOT`: https://github.com/JFormDesigner/FlatLaf#snapshots
